cmd/influx/pkg.go

```diff
@@ -108,11 +109,32 @@ func (b *cmdPkgBuilder) cmdPkgApply() *cobra.Command {
 
 func (b *cmdPkgBuilder) pkgApplyRunEFn() func(*cobra.Command, []string) error {
 	return func(cmd *cobra.Command, args []string) (e error) {
+		if b.orgID == "" && b.org == "" {
+			return fmt.Errorf("must specify org-id, or org name")
+		} else if b.orgID != "" && b.org != "" {
+			return fmt.Errorf("must specify org-id, or org name not both")
+		}
 		color.NoColor = !b.hasColor
+		var influxOrgID *influxdb.ID
 
-		influxOrgID, err := influxdb.IDFromString(b.orgID)
-		if err != nil {
-			return fmt.Errorf("invalid org ID provided: %s", err.Error())
+		if b.orgID != "" {
+			var err error
```

Comment:
this isn't going to work, breaks the pattern for the builder...

Reply:
any SVC dep needs to be provided through DI not dumped in here globally

needs fixup

Reply:
Thanks for catching, we'll fix in the morning.